x-pack/qa/smoke-test-index-lifecycle-with-security/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'elasticsearch.standalone-rest-test'
+apply plugin: 'elasticsearch.rest-test'
+
+dependencies {
+  testCompile project(path: xpackProject('plugin').path, configuration: 'testArtifacts')
+}
+
+// bring in machine learning rest test suite
+task copyILMRestTests(type: Copy) {
+  into project.sourceSets.test.output.resourcesDir
+  from project(xpackProject('plugin').path).sourceSets.test.resources.srcDirs
+  include 'rest-api-spec/test/index_lifecycle/**'
+}
+
+integTestRunner {
+  systemProperty 'tests.rest.cluster.username', System.getProperty('tests.rest.cluster.username', 'test_user')
+  systemProperty 'tests.rest.cluster.password', System.getProperty('tests.rest.cluster.password', 'x-pack-test-password')
+}
+
+integTestCluster {
+  dependsOn copyILMRestTests
+  setting 'xpack.index_lifecycle.enabled', 'true'
+  setting 'xpack.security.enabled', 'true'
+  setting 'xpack.watcher.enabled', 'false'
+  setting 'xpack.monitoring.enabled', 'false'
+  setting 'xpack.ml.enabled', 'false'
+  setting 'xpack.license.self_generated.type', 'trial'
+  setupCommand 'setupDummyUser',
+               'bin/elasticsearch-users',
+               'useradd', System.getProperty('tests.rest.cluster.username', 'test_user'),
+               '-p', System.getProperty('tests.rest.cluster.password', 'x-pack-test-password'),
+               '-r', 'superuser'
+  waitCondition = { node, ant ->
+    File tmpFile = new File(node.cwd, 'wait.success')
+    ant.get(src: "http://${node.httpUri()}/_cluster/health?wait_for_nodes=>=${numNodes}&wait_for_status=yellow",
+            dest: tmpFile.toString(),
+            username: 'test_user',
```

Comment:
Would try to get this from the system properties as well. 
Maybe extract these into variables to remove repetition.
I'm assuming the properties are there so they can be used with external clusters  for which the wait conditions are not called, but that's an implementation detail  and looks odd not to consider the properties in here.